src/transformers/models/timm_backbone/modeling_timm_backbone.py

```diff
@@ -72,6 +72,11 @@ def __init__(self, config, **kwargs):
             out_indices=out_indices,
             **kwargs,
         )
+
+        # Converts all `BatchNorm2d` and `SyncBatchNorm` or `BatchNormAct2d` and `SyncBatchNormAct2d` layers of provided module into `FrozenBatchNorm2d` or `FrozenBatchNormAct2d` respectively
+        if hasattr(config, "freeze_batch_norm_2d") and config.freeze_batch_norm_2d is True:
+            self._backbone = timm.layers.freeze_batch_norm_2d(self._backbone)
```

Comment:
Rather than have this hard coded within the `__init__` it would be good to have both the freeze and unfreeze methods than can be applied to the model as-needed i.e.

```
class TimmBackbone(PreTrainedModel, BackboneMixin):
    def __init__(...):
        if getattr(config, "freeze_batch_norm_2d", False):
            self.freeze_batch_norm_2d()

    def freeze_batch_norm_2d(self):
        self._backbone = timm.layers.freeze_batch_norm_2d(self._backbone)

    def unfreeze_bach_norm_2d(self):
        timm.layers.unfreeze_batch_norm_2d(self._backbone)
```

Reply:
Done! 
I like the idea of having our own freeze and unfreeze methods. :)
I tested and noticed that `freeze_batch_norm_2d(...)` and `unfreeze_batch_norm_2d(...)` are both done in place, so no need to return.